static.js

```diff
@@ -4,6 +4,6 @@ const handler = require('serve-handler');
 
 require('http').createServer(function(req, res) {
   handler(req, res, { public: '.' }).catch(err => res.statusCode(500).send(err.message));
-}).listen(8089);
+}).listen(8088);
```

Comment:
@vkarpov15 Changed this 7 months ago to 8089, this probably affects how https://mongoosejs.com is served, potentially a breaking change.

Reply:
reverted it, also made it possible to set the port like

`$ PORT=8080 node static`